### PR TITLE
Update Bones_influences_per_vertex.md

### PR DESCRIPTION
**Pull request by dad72**
Adding a more complete picture and for the French version of 3ds max

### CODE REVIEW

File: content/exporters/3DSMax/Bones_influences_per_vertex.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ This is because Babylon.js support a maximum of 4 bones influences per vertex.
 
 Here is how to solve this in 3ds max (By default 3ds max has a limit of 20):
 
-![bones influences parameter](http://www.visualiser.fr/img/max.jpg)
+In 3ds max version English:
```
Comment (deltakosh) [inline content/exporters/3DSMax/Bones_influences_per_vertex.md]: In 3ds Max (English version)

File: content/exporters/3DSMax/Bones_influences_per_vertex.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ This is because Babylon.js support a maximum of 4 bones influences per vertex.
...
 
-Simply add '4' in the red part of the image and you're done!
\ No newline at end of file
+![bones influences parameter En](http://www.visualiser.fr/img/max.jpg)
+
+In 3ds max version French:
```
Comment (deltakosh) [inline content/exporters/3DSMax/Bones_influences_per_vertex.md]: In 3ds Max (French version)

**Comment by dad72** [comment]
I should change?

**Comment by deltakosh** [comment]
Please :)

**Pull request closed without merging**